docs/devguide/metricset-details.asciidoc

```diff
@@ -204,13 +205,99 @@ waitFor ${REDIS_HOST} ${REDIS_PORT} Redis
 The environment expects your service to be available as soon as it receives a response from
 the given address and port.
 
+[float]
+===== Adding the standard metricset integration tests
+
+There are normally two integration tests that are part of every metricset: `TestFetch` and `TestData`.
+Both tests will start up a new instance of your metricset and fetch an event. In order to start a metricset, you need to create a configuration object:
+
+[source,go]
+----
+func getConfig() map[string]interface{} {
+	return map[string]interface{}{
+		"module":           "{module}",
+		"metricsets":       []string{"{metricset}"},
+    "hosts":      []string{GetEnvHost() + ":" + GetEnvPort()},
```

Comment:
your alignment looks off here.

Reply:
That's odd, doesn't show up that way in vscode....